src/rain-report/dto/create-rain-report.dto/create-rain-report.dto.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { validate } from 'class-validator';
```

Comment:
Specs for `CreateRainReportDto` to ensure that it requires a boolean `rain` value. This DTO is used to validate incoming payloads for the `POST /api/data` action to create a new `RainReport` record.